python_modules/dagit/dagit/schema.py

```diff
@@ -33,19 +37,140 @@ def results_or_errors(root_obj, fn, info, *argv):
         return [PythonError(*error)]
     try:
         return fn(root_obj, info, *argv)
-    except:
+    except Exception:  # pylint: disable=broad-except
         return [PythonError(*sys.exc_info())]
 
 
+def non_null_list(ttype):
+    return graphene.NonNull(graphene.List(graphene.NonNull(ttype)))
+
+
+class ComputeNodeGraph(graphene.ObjectType):
```

Comment:
Probably needs at least a backreference to pipeline, maybe in future to config that was used to create it.

Reply:
✔️